tests/core/test_oauth_clients.py

```diff
@@ -0,0 +1,25 @@
+from flask import json
+from newsroom.tests.users import test_login_succeeds_for_admin, init as user_init  # noqa
+from superdesk import get_resource_service
+
+
+def test_oauth_clients(client):
+    test_login_succeeds_for_admin(client)
+    # Register a new client
+    response = client.post('/clients/new', data=json.dumps({
+        'name': 'client1',
+    }), content_type='application/json')
+
+    assert response.status_code == 201
```

Comment:
would be good to assert the client secret is there and then use it to generate auth token